src/main/java/gov/usgs/wma/waterdata/data/DataController.java

```diff
@@ -105,8 +106,12 @@ public void getTimeSeries(
 			rtn = ogc404Payload;
 		}
 
-		if(!outputStreamed) {
-			response.getWriter().print(rtn);
+		if (!outputStreamed) {
+			if (streamingOutput) {
+				response.getOutputStream().print(rtn);
+			} else {
+				response.getWriter().print(rtn);
+			}
```

Comment:
I bet you could wrap the output stream into a writer so that you only have one interface to manage?

Reply:
Yes, or just get the json output using response.getOutputStream() .. There was a character encoding issue in 
the tests that needs resolving first.